### PR TITLE
Update matplotlib and cartopy notebook

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
The plotting updates extracted from https://github.com/jorisvandenbossche/course-python-geospatial/pull/3

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
VMM data is still missing I think

File: notebooks/visualization-01-matplotlib.md [hunk 1]
```diff
@@ -61,15 +61,33 @@ On its own, drawing the figure artist is uninteresting and will result in an emp
...
+![](../img/matplotlib_fundamentals.png)
+
 There is no limit on the number of Axes artists which can exist on a Figure artist. Let's go ahead and create a figure with a single Axes artist, and show it using pyplot:
 
 ```{code-cell} ipython3
 ax = plt.axes()
 ```
 
+```{code-cell} ipython3
+fig.set_t
```
Comment (jorisvandenbossche) [inline notebooks/visualization-01-matplotlib.md:73]: Was this added on purpose?

File: notebooks/visualization-01-matplotlib.md [hunk 2]
```diff
@@ -61,15 +61,33 @@ On its own, drawing the figure artist is uninteresting and will result in an emp
 
 By far the most useful artist in matplotlib is the **Axes** artist. The Axes artist represents the "data space" of a typical plot, a rectangular axes (the most common, but not always the case, e.g. polar plots) will have 2 (confusingly named) **Axis** artists with tick labels and tick marks.
 
+![](../img/matplotlib_fundamentals.png)
```
Comment (jorisvandenbossche) [inline notebooks/visualization-01-matplotlib.md:64]: I add this figure (copied from DS-python-data-analysis)

File: notebooks/visualization-01-matplotlib.md [hunk 3]
```diff
@@ -61,15 +61,33 @@ On its own, drawing the figure artist is uninteresting and will result in an emp
...
+```{code-cell} ipython3
+type(ax)
+```
+
+```{code-cell} ipython3
+type(ax.xaxis), type(ax.yaxis)
+```
+
+```{code-cell} ipython3
+ax.xaxis.set_tic
```
Comment (jorisvandenbossche) [inline notebooks/visualization-01-matplotlib.md:85]: same here?

File: notebooks/visualization-03-cartopy.md [hunk 4]
```diff
@@ -408,6 +408,69 @@ sst.plot(ax=ax, transform=ccrs.PlateCarree(),
...
++++
+
+A lot of online map services are available, which are capable of providing background layers to a map similar as these are used in online applications. The [OGC WMS/WMTS](https://www.ogc.org/standards/wms) are important standards by which a lot of map layers are made available. 
+
+A (non-official, but very comprehensive) list of services is made available by Michel Stuyts, see https://wms.michelstuyts.be. It is always a matter of finding the correct URL and the `layer_name` for a given service. 
+
+As an example, consider the Orthophoto WMS (provided by the National Geographic Institute): 
+
+- WMS: https://wms.ngi.be/inspire/ortho/service
+- GetCapabilities: https://wms.ngi.be/inspire/ortho/service?REQUEST=GetCapabilities&SERVICE=WMS
```
Comment (jorisvandenbossche) [inline notebooks/visualization-03-cartopy.md:422]: Clicking those links will open a file, that's expected? (they don't have a web client showing the result of a query?)

Comment (stijnvanhoey) [reply notebooks/visualization-03-cartopy.md:422]: Indeed, it is just as this service provides it, we'll have to deal with it as such. I would keep it here?

Comment (jorisvandenbossche) [reply notebooks/visualization-03-cartopy.md:422]: Yep

**Pull request merged**